CHANGELOG.md

```diff
@@ -5,6 +5,10 @@
 - Add detection support for Rails 8 (https://github.com/heroku/heroku-buildpack-ruby/pull/1498)
 - Support Node.js on ARM builds (https://github.com/heroku/heroku-buildpack-ruby/pull/1499)
 
+## [v280] - 2024-11-04
```

Comment:
```suggestion
## [v281] - 2024-11-04
```

Reply:
You already have a `v280` entry in the changelog.

Reply:
Actually, you have two `v280` entries in your changelog already. And looking at the buildpack registry the `heroku/ruby` buildpack is at `v299`. Is there some changelog cleanup that needs to happen here?

Reply:
This PR is pure ceremony to get approval to release the binary. I am not going to release main or rev the buildpack version.

Reply:
Sure, but some changelog cleanup does seem warranted. If not in this PR then that's fine but it's confusing to see.